### DISCUSSION TITLE
Using lighthouse cli with firefox as a headless browser

### DISCUSSION THREAD

**Discussion by thearunkumar** [GoogleChrome/lighthouse#13987] [Q&A]
Hi

I'm trying to run lighthouse cli and measure the performance and while it works fine with `chrome-launcher`, I am actually looking for running using a firefox headless browser mode as the application runs mainly on firefox.

Is there a way to achieve that using `karma-firefox-launcher` or something similar?

Any guidance is appreciated.

Thanks,

**Comment by adamraine** [comment]
No. Lighthouse can only test pages using Chrome.

**Reply by thearunkumar** [reply]
Even with `puppeteer-firefox`, it doesn't work?

**Reply by adamraine** [reply]
Lighthouse requires a Chrome DevTools protocol connection to analyze to the page. We can't use Firefox's protocol even with puppeteer.
